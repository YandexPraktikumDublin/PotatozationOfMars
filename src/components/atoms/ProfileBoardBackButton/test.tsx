import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ProfileBoardBackButton } from '.'

describe('<ProfileBoardBackButton />', () => {
  it('should renders correct <ProfileBoardBackButton />', () => {
    const onClick = jest.fn()

    const wrapper = shallow(<ProfileBoardBackButton onClick={onClick} />)

    wrapper.simulate('click')

    expect(onClick).toHaveBeenCalledTimes(1)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
