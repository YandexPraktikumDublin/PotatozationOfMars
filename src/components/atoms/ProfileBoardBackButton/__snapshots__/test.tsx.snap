// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileBoardBackButton /> should renders correct <ProfileBoardBackButton /> 1`] = `
<button
  className="absolute top-0 left-0"
  onClick={
    [MockFunction] {
      "calls": Array [
        Array [],
      ],
      "results": Array [
        Object {
          "type": "return",
          "value": undefined,
        },
      ],
    }
  }
>
  <svg
    height="20"
    width="20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      className="fill-primary dark:fill-white"
      d="M19.377 9.44H2.042l5.186-5.166a.622.622 0 10-.881-.881L.18 9.559a.63.63 0 000 .881l6.167 6.167a.622.622 0 10.88-.88l-5.185-5.041h17.335a.623.623 0 000-1.246z"
    />
  </svg>
</button>
`;
